log.go

```diff
@@ -12,19 +16,19 @@ const (
 )
 
 type Log struct {
-	ExecutionID  string
-	Name         string
-	Type         string
-	Time         time.Time
+	ExecutionID  string    `gorm:"index;type:varchar(255)"`
```

Comment:
I have some doubts for keeping gorm tags in general Log struct, because not all users use gorm for storing logs.
What do you think if move them to gorm package and convert general Log struct to gorm one somewhere in gorm related code?